modules/microshift-viewing-audit-logs.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//microshift_troubleshooting/microshift-audit-logs.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations through audit logs
+
+You can identify pod security admission violations on a workload by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find pod security admission violation in a workload.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To view the audit logs, run the following command: 
```

Comment:
Wouldn't this technically be two commands, and wouldn't the first return something (a node name?)

So wouldn't step 1 be like "Enter the following command to retrieve the node name"

And step 2 be like "Enter the following command to view the audit logs of the node" 

?

Reply:
Ooo yes great catch